### ISSUE TITLE
Odo 0.4.0 complaining about windows-style line terminators with csv->postgresql

### ISSUE THREAD

**Issue by aktiur** [blaze/odo#403]
As of 0.4.0, Odo throws a ValueError whenever importing a CSV file with windows style line terminators into PostgreSQL (with error message: "PostgreSQL does not support line terminators other than \n").

However, according to PostgreSQL 9.5 COPY FROM's documentation, "COPY FROM can handle lines ending with newlines, carriage returns, or carriage return/newlines."

Having used Odo 0.3.4 before, I did not have any trouble importing such CSV files.

Therefore, I don't understand why this check and error were introduced?

Sincerely,
Arthur Cheysson

**Comment by llllllllll** [comment]
I think this check might have been overly conservative. I was able to reproduce this locally and removing our guard does not seem to cause any problems writing a dos style terminator to postgres.

@cpcloud you added this check pretty recently in 295178a63ea438339d3952847fe3f90f3a92e062, do you think it is safe to remove?

**Comment by pratapvardhan** [comment]
I can confirm this error on Windows machine with Odo `0.4.0`

**Comment by sanand0** [comment]
Agreed -- the check does seem conservative. On my Windows machine, it works fine on PostgreSQL 9.4 without the check.

**Comment by pratapvardhan** [comment]
Any thoughts on removing the check?

**Comment by a-muratov** [comment]
Still hitting this on Windows with Odo `0.5.0`.

**Comment by binsi** [comment]
Can still confirm this happening on Windows & odo-0.5.0
